src/running/command/runbms.py

```diff
@@ -196,6 +196,8 @@ def hz_to_ghz(hzstr: str) -> str:
 def get_log_prologue(runtime: Runtime, bm: Benchmark) -> str:
     output = "\n-----\n"
     output += "mkdir -p PLOTTY_WORKAROUND; timedrun; "
+    # Chrome workaround to remove the cache directory between iterations.
+    output += "rm -rf /tmp/chrome-expr-cache; "
```

Comment:
This doesn't actually run anything. It gets added to the log file, but that's it.

Reply:
Note that running has already created a temporary directory for each run.

https://github.com/anupli/running-ng/blob/815cf54d96b3b50e9c1b3001e63dd6292bd51373/src/running/command/runbms.py#L391C59-L391C69

So you might be able to create a subfolder under it as the temporary directory for Chrome.

Reply:
So what might be the cleanest is to use the plugin API to create the temporary folder under `runbms_dir` for Chrome to use. The plugin object will be created by an instance of `Chrome`, and will only run when `Chrome` is used for example.

The plugin object can also delete the folder when the benchmark finishes.